frontend/src/App.js

```diff
@@ -2,65 +2,113 @@ import SearchPage from "./search";
 import ProfessorList from "./professorList.js";
 import ProfessorPage from "./professorPage";
 import { useState, useEffect } from 'react';
-
-import Professor from "./Professor";
 import React from "react";
 
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
-import Course from "./Course";
 import axios from "axios";
-import Review from "./Review";
 
 function App(props){
    const [searchedProfessors, getSearchedProfessors] = useState([]);
-   const [searchBy, getSearchBy] = useState("");
-   const [searchWith, getSearchWith] = useState("");
    const [professor, getProfessor] = useState(null);
+   const [searchBy, getSearchBy] = useState("nothin'");
```

Comment:
What does useState("nothin'") do? Can you change it to look slightly more professional?

Reply:
Oop! I meant to use that as a testing value. I will change that.